wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
So some of the `get_global_param_string` can require up to 1606 characters, so quite way more than 254 (255 minus the last one being the terminating 0). I did not knew that, and it has some importance.
This may in deed be in favour to have long enough buffer to handle such long outputs.
In an other way, as only the pointer is passed to the library, keeping the single buffer but larger will just slow by a little the loading of the library, have no impact on the functions execution, and allowing any output message (up to this size of course).
So for me the best compromise for Julia’s wrapper would be to set the buffer length to 1607. What's your opinion about that?